discussions/utils.py

```diff
@@ -60,3 +63,29 @@ def get_token_for_request(request, force_create=False):
         str: the token or None
     """
     return get_token_for_user(request.user, force_create=force_create)
+
+
+def email_mm_admin_about_stale_memberships(membership_ids):
+    """
+    Email list of stale memberships to mm admin
+
+    Args:
+        membership_ids(List): List of ids
+    """
+    if getattr(settings, 'ADMIN_EMAIL', None) is None:
+        raise ImproperlyConfigured('Setting ADMIN_EMAIL is not set')
+
+    subject = "List of stale memberships"
```

Comment:
Suggest changing this to "Alert: {num_not_synced} MicroMasters channel memberships not synced" or something like that so the recipient knows what the email is about at a glance